### PR TITLE
FeeShare: Authz payout support

### PR DESCRIPTION
**Pull request by Reecepbcups**

### CODE REVIEW

File: interchaintest/helpers/cosmwasm.go [hunk 1]
```diff
@@ -92,3 +92,91 @@ func ExecuteMsgWithFee(t *testing.T, ctx context.Context, chain *cosmos.CosmosCh
 		t.Fatal(err)
 	}
 }
+
+func ExecuteAuthzGrantMsgWithFee(t *testing.T, ctx context.Context, chain *cosmos.CosmosChain, granter ibc.Wallet, grantee ibc.Wallet, contractAddr, amount, feeCoin, message string) {
```
Comment (Reecepbcups) [inline interchaintest/helpers/cosmwasm.go:96]: Future PR but we should abstract these away from hardcoded msg types, but out of scope here

Comment (Reecepbcups) [reply interchaintest/helpers/cosmwasm.go]: update nvm I did it now as there were some unused args already

**Review by Reecepbcups — COMMENTED** [review]
LGTM

**Review by joelsmith-2019 — APPROVED** [review]
Looks good to me as well!

**Pull request merged**